src/keys.c

```diff
@@ -363,6 +363,8 @@ load_keys(const char* jwkdir)
         }
 
         if (strcmp(dot, pattern) == 0) {
+            json_error_t error;
```

Comment:
Could not this be moved closer to its usage? If no error, this declaration is not required, so I would remove the declaration to block inside if(!json) { ...

Reply:
You always have a new enough C compiler? I'll move assuming yes.